app.py

```diff
@@ -0,0 +1,87 @@
+from flask import Flask, request
+from harvester.database.interface import HarvesterDBInterface
+from harvester.database import init_db
+from tests.database.data import new_source, new_job, new_error
+
+app = Flask(__name__)
+db = HarvesterDBInterface()
+
+@app.route('/', methods=['GET'])
+def index():
+    html = "<b>" + init_db.create_tables() + "</b>"
+    html += "<ul>"
+    for rule in app.url_map.iter_rules():
+        if 'static' not in rule.endpoint and 'index' not in rule.endpoint:
+            html += (f"<li>{rule.endpoint} : "
+                     f"<a href='{rule.rule}'>{rule.rule}</a></li><br>")
+    html += "</ul>"
+    return html
+
+@app.route('/add_source', methods=['GET'])
+def add_harvest_source():
+    source=db.add_harvest_source(new_source)
+    return(f"Added new source with ID: {source.id}")
+
+@app.route('/add_job', methods=['GET'])
+def add_harvest_job():
+    source_id = request.args.get('source_id', None)
+    if source_id is None:
+        return 'Please provide source_id: /add_job?source_id=xxx'
+    else:
+        job=db.add_harvest_job(new_job, source_id)
+        return(f"Added new job with ID: {job.id}")
+
+@app.route('/add_error', methods=['GET'])
+def add_harvest_error():
+    job_id = request.args.get('job_id', None)
+    if job_id is None:
+        return 'Please provide job_id: /add_error?job_id=xxx'
+    else:
+        err=db.add_harvest_error(new_error, job_id)
+        return(f"Added new error with ID: {err.id}")
```

Comment:
I don't think this should exist; however that's my opinion and would default to how others feel about this. I think we should never be manually creating errors through the API, it should be happening by the runner directly into the database.

Reply:
100% agree with that.

Reply:
agree. I just list here for initial manually testing database. Will remove it later.